meinberlin/apps/budgeting/assets/FilterBar.jsx

```diff
@@ -71,24 +71,22 @@ export const FilterBar = props => {
         <FilterBarListMapSwitch query={filterString} />
       </div>
       <div className="offset-lg-2 col-lg-8">
-        <div className="filter-bar__section">
-          <div className="filter-bar__section-item">
+        <div className="control-bar control-bar-search">
+          <div className="searchinput">
```

Comment:
these are equivalent `sub-wrappers` to `dropdown`, for the search input and the filters toggle button.

Reply:
> these are equivalent `sub-wrappers` to `dropdown`, for the search input and the filters toggle button. So nested in `control-bar`, we want to have either `dropdown`, `searchinput` or `togglebutton`.
> 
> maybe instead of adding two new specific, it is better to rename `dropdown` to something more general? `control-bar-item`?

not sure I follow, you want to rename `dropdown` class? or just the wrapper classes? I would be in favour of having a general wrapper class but we should do it everywhere so any instance of a control bar should be updated so we are only maintaining 1 style and delete the old code.